core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -1567,53 +1567,54 @@ class KafkaApis(val requestChannel: RequestChannel,
     }
   }
 
-  def handleDescribeGroupRequest(request: RequestChannel.Request): Unit = {
+  def handleDescribeGroupsRequest(request: RequestChannel.Request): Unit = {
+    val describeRequest = request.body[DescribeGroupsRequest]
+    val includeAuthorizedOperations = describeRequest.data.includeAuthorizedOperations
 
-    def sendResponseCallback(describeGroupsResponseData: DescribeGroupsResponseData): Unit = {
-      def createResponse(requestThrottleMs: Int): AbstractResponse = {
-        describeGroupsResponseData.setThrottleTimeMs(requestThrottleMs)
-        new DescribeGroupsResponse(describeGroupsResponseData)
-      }
-      requestHelper.sendResponseMaybeThrottle(request, createResponse)
+    def sendResponse(response: AbstractResponse): Unit = {
+      requestHelper.sendResponseMaybeThrottle(request, requestThrottleMs => {
```

Comment:
did we want a trace log here too?

Reply:
added it.

Reply:
Did we want to refactor to include the new SendResponse method?

Reply:
Yes, will do when that PR is merged.